server/Cargo.toml

```diff
@@ -2,8 +2,12 @@
 name = "server"
 version = "0.1.0"
 edition = "2021"
+default-run = "server"
```

Comment:
this lets you run `cargo run` from the workspace root and have the server start.